etc/testing/circle/run_api_performance_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+pachctl_tag=$PACHD_VERSION 
+if [ "$PACHD_VERSION" == "latest" ]; then
+    pachctl_tag="v2.5.0-alpha.1" #$(git tag --sort=taggerdate | tail -1) Hard code for the moment until nightly docker images are deploying
```

Comment:
would be nice to be configurable version?

Reply:
It should be using that latest tag code, this was a temp thing. Do you know if/when the images are going to be published to dockerhub in the future? That's the real blocker here.

Reply:
It deployed last night, so this should be good to go! I removed the stop gap.